### PR TITLE
feat: Add intro page

### LINKED ISSUE #276: Logotype
The site is a bit forgettable, it doesn't have a name anywhere, compare this to other tools like: https://bundlephobia.com/package/webext-tools@2.0.1

I think we can have a small logotype (logo made of text)  to show somewhere above the search field, similar to Node.js' and npm's

I didn't find either one of those fonts, but I was thinking we could use another blocky font like them. There are [many on dafont](https://www.dafont.com/theme.php?cat=301&page=1&psize=xs&text=npmgraph) but most of them are pretty bad, especially at small sizes.

Examples:

-  [link](https://www.dafont.com/tt-octosquares.font?text=npmgraph&psize=s)
-   [link](https://www.dafont.com/robot-2.font?text=npmgraph)
-   [link](https://www.dafont.com/tt-mussels.font?text=npmgraph&fpp=200&psize=s) 
-   [link](https://www.dafont.com/classic-robot.font?fpp=200&psize=s&text=npmgraph)

But there are many better. This issue is to explore the idea

Related:

- https://github.com/npmgraph/npmgraph/issues/277

### LINKED ISSUE #277: Add homepage
For example:

Screenshot

Since npmgraph is a SPA, it wouldn't be too complicated to create something like that using the contents of the Start tab + a larger logo.

Related:

- https://github.com/npmgraph/npmgraph/issues/276

### PR DESCRIPTION
**Pull request by fregante**
Barebones, it just needs a better logo:

- #276 

Screenshot

After #240 is merged, I can extract the instructions to their own component and reuse them (rather than copy-paste them)

### CODE REVIEW

**Comment by broofa** [comment]
Dude, this is sweet!  And that octocat-wave animation...



> After 240 is merged

Did you mean 284?  240 isn't a PR.

**Comment by fregante** [comment]
> Dude, this is sweet! And that octocat-wave animation…

https://github.com/tholman/github-corners

image



> > After 240 is merged
> 
> Did you mean 284? 240 isn't a PR.

Tomato tomato 😃 

But yes

If you have any suggestions let me know. I'll merge it after after I create the logotype SVG.

**Comment by fregante** [comment]
Screen Shot 2024-08-10 at 02 12 29
Screen Shot 2024-08-10 at 02 12 08

**Review by broofa — APPROVED** [review]
Looks good to me!

**Comment by fregante** [comment]
Follow-up tasks, at some point:

- [ ] Consolidate input help component (presumably only keep those 3 points on the intro, not the package.json stuff)
- [ ] Make sure that GraphViz is preloaded on this intro page (I presume it's not)

**Comment by foxt** [comment]
slight issue with this: there's no support for dragging package.json files into the start page, currently you need to enter something into the box to get into the app, then you can drag your package.json in

File: components/App/App.tsx [hunk 1]
```diff
@@ -3,10 +3,17 @@ import type LoadActivity from '../../lib/LoadActivity.js';
 import GraphDiagram from '../GraphDiagram/GraphDiagram.js';
 import Inspector from '../Inspector.js';
 import './App.scss';
+import { useQuery } from '../../lib/useQuery.js';
+import Intro from '../Intro.js';
 import { Loader } from './Loader.js';
 
 export default function App() {
   const activity = useActivity();
```
Comment (fregante) [inline components/App/App.tsx:11]: @foxt nice catch! I think it's a matter of moving `useExternalInput` from `Inspector.tsx` to this line here. PR welcome

**Comment by fregante** [comment]
@foxt it works now

**Pull request merged**